recipes/pcraster/conda_build_config.yaml

```diff
@@ -0,0 +1,15 @@
+c_compiler:    # [win]
+  - vs2017     # [win]
+  - vs2017     # [win]
+
+cxx_compiler:  # [win]
+  - vs2017     # [win]
+  - vs2017     # [win]
+
+vc:            # [win]
+  - 14.1       # [win]
+  - 14.1       # [win]
+
+python:
+  - 3.6
+  - 3.7
```

Comment:
I am not sure you need this file

Reply:
You mean removing the entire build config file? I can try that. Or do you only refer to the python part there? thanks

Reply:
Yeah, the entire build config file